schemas/book.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE catalog_of_my_things;
+
+CREATE TABLE Book (
```

Comment:
In the task card, there's a requirement for a books table: ' books table (add all properties and associations from the parent Item class as table columns)'
Kindly add source_id to the table like shown in the `item` properties

![revispencer](https://github.com/kwesiObuobi/catalog-group-project/assets/84572891/14ee4a35-3938-4c73-801f-5856310237bd)